changelogs/fragments/import_log_issues.yaml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
```

This shouldn't require a minor version bump. Only a patch
```suggestion
bugfixes:
```